syntax = "proto3";

package com.collaborative.ml;

message PB_Keys {
    int32 public_key_s = 1;
    int32 public_key_w = 2;
    int32 public_key_l = 3;
    repeated PB_PublicN public_key_n = 4;
    string public_key_g = 5;
    string public_key_delta = 6;

    int32 auth_server_i = 7;
    string auth_server_si = 8;
}

message PB_PublicN {
    string n = 1;
}